test/extended/cli/idle.go

```diff
@@ -0,0 +1,148 @@
+package cli
+
+import (
+	"fmt"
+	g "github.com/onsi/ginkgo"
+	o "github.com/onsi/gomega"
+	exutil "github.com/openshift/origin/test/extended/util"
+	"k8s.io/apimachinery/pkg/util/wait"
+	admissionapi "k8s.io/pod-security-admission/api"
+	"path/filepath"
+	"strings"
+	"time"
+)
+
+const (
+	idledAnnotation     = "idling.alpha.openshift.io/idled-at"
+	prevScaleAnnotation = "idling.alpha.openshift.io/previous-scale"
+	scaledReplicaCount  = "2"
+)
+
+var _ = g.Describe("[sig-cli] oc idle can idle the scalable controllers associated with the services [apigroup:apps.openshift.io][apigroup:route.openshift.io][apigroup:project.openshift.io][apigroup:image.openshift.io]", func() {
```

Comment:
Apps for DC, route for Route, Project for `oc project`, image for `image-registry.openshift-image-registry.svc:5000/openshift/tools:latest`

Reply:
I'd suggest keeping the describe sentence as short as possible, since it's being added to all `g.It` invocations. How about just `[sig-cli] oc idle [apigroup:apps.openshift.io][apigroup:route.openshift.io][apigroup:project.openshift.io][apigroup:image.openshift.io]`